tests/DoctrineMongoODMModuleTest/Doctrine/CliTest.php

```diff
@@ -26,7 +34,7 @@
  * @link    http://www.doctrine-project.org/
  * @author  Adam Homsi <adam.homsi@gmail.com>
  */
-class CliTest extends PHPUnit_Framework_TestCase
+final class CliTest extends \PHPUnit_Framework_TestCase
```

Comment:
@gianarb technically, this is a BC break, but it's acceptable since this library hasn't reached 1.0.0 yet. If you're going to merge this in, please add a BC break notice. People shouldn't be using this class anyways, but better play it safe.